generators/client/templates/angular/_package.json

```diff
@@ -151,12 +150,8 @@
     "webpack:prod": "<%= clientPackageManager %> run cleanup && <%= clientPackageManager %> run webpack:prod:main && <%= clientPackageManager %> run clean-www",
     "webpack:test": "<%= clientPackageManager %> run test",
     "webpack-dev-server": "node --max_old_space_size=4096 node_modules/webpack-dev-server/bin/webpack-dev-server.js",
-    "webpack": "node --max_old_space_size=4096 node_modules/webpack/bin/webpack.js",
-    <%_ if (protractorTests) { _%>
+    "webpack": "node --max_old_space_size=4096 node_modules/webpack/bin/webpack.js"<% if (protractorTests) { %>,
```

Comment:
I think the `<%_` variant of template is more readable

Reply:
Agree with this. I had to use the other way to deal with trailing commas in the JSON. Do you know another way to do it.